README.md

```diff
@@ -169,17 +159,15 @@ $ pnpm -w start:electron
 
 > `-w` means workspace root package, with this you don't need to have your current working directory at the repo-root to run those scripts.
 
-For development with local deltachat-core read the [docs](docs/UPDATE_CORE.md)
+For development with local `deltachat-core-rust` read [`docs/UPDATE_CORE.md`](docs/UPDATE_CORE.md).
 
 ### Troubleshooting <a id="troubleshooting"></a>
 
-This module builds on top of [`deltachat-core-rust`](https://github.com/deltachat/deltachat-core-rust), which in turn has external dependencies. The instructions below assume a Linux system (e.g. Ubuntu 18.10).
-
-Read the error, maybe it already tells you what you need to do. If not feel free to file an issue in this github repo.
-
-Make sure that your nodejs version is `20.0.0` or newer.
-
-If you still get errors look at the instructions in [UPDATE_CORE Readme](./docs/UPDATE_CORE.md) to set things up [`or write an issue`](https://github.com/deltachat/deltachat-desktop/issues/new/choose).
+* This module builds on top of [`deltachat-core-rust`](https://github.com/deltachat/deltachat-core-rust),
+which in turn has external dependencies. The instructions below assume a Linux system (e.g. Ubuntu 18.10).
+* Read the error, maybe it already tells you what you need to do. If not feel free to file an issue in this github repo.
+* Make sure that your nodejs version is `20.0.0` or newer.
+* If you still get errors look at the instructions in [`docs/UPDATE_CORE.md`](docs/UPDATE_CORE.md) to set things up or [write an issue](https://github.com/deltachat/deltachat-desktop/issues/new/choose).
```

Comment:
Unordered lists again. Also some markdown line breaks. I forget what the format is called again that Holga likes so much :)

Reply:
semantic line breaks?

Reply:
That's the one!